app/models/SkillMatrixItem.scala

```diff
@@ -0,0 +1,17 @@
+package models
+
+import models.EnumTypes.SkillLevel.SkillLevel
+import models.EnumTypes.SkillType.SkillType
+import play.api.libs.functional.syntax._
+import play.api.libs.json.{JsPath, Reads}
+
+/**
+  * Created by tatianamoldovan on 10/02/2017.
```

Comment:
We usually suggest to remove this, because git knows more than who created etc. already.